src/Http/Http.Extensions/src/RequestDelegateFactory.cs

```diff
@@ -1211,17 +1211,19 @@ private static void SetPlaintextContentType(HttpContext httpContext)
         private static string BuildErrorMessageForMultipleBodyParameters(FactoryContext factoryContext)
         {
             var errorMessage = new StringBuilder();
-            errorMessage.Append($"Failure to infer one or more parameters.\n");
-            errorMessage.Append("Below is the list of parameters that we found: \n\n");
-            errorMessage.Append($"{"Parameter",-20}|{"Source",-30} \n");
-            errorMessage.Append("---------------------------------------------------------------------------------\n");
+            errorMessage.AppendLine("Failure to infer one or more parameters.");
```

Comment:
@halter73 \ @davidfowl was this intentionally using `\n` versus `AppendLine`?

Reply:
I don't believe so unless @rafikiassumaniMSFT says otherwise.